ods_ci/tests/Resources/RHOSi.resource

```diff
@@ -70,6 +70,7 @@ Protect Sensitive Variables In Keywords
 Initialize Global Variables
     [Documentation]   Sets global variables
     [Arguments]    ${force_fetch}=False
+    Assign Vars According To Product    ${PRODUCT}
```

Comment:
${PRODUCT} is already a global variable so you don't need this argument

Reply:
I had to include it as an input param because of:
`Validate Trusted CA Bundles State Managed :: The purpose of this t... | FAIL |
Parent suite setup failed:
Keyword 'RHOSi.Assign Vars According To Product' expected 1 argument, got 0.`

Reply:
Removed